cloudsync/exceptions.py

```diff
@@ -29,6 +29,10 @@ class CloudRootMissingError(CloudTemporaryError):     # ENOENT, but treated diff
     pass
 
 
+class CloudResourceModifiedError(CloudTemporaryError):  # resource being updated changed since the caller last read it
```

Comment:
Please add a test that gets coverage on this, demonstrating it is treated like a temporarary error

Reply:
done.